io.gitlab.cyberphantom52.sudoku_solver.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id" : "io.gitlab.cyberphantom52.sudoku_solver",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "sudoku-solver",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
+              {
+                "type": "git",
+                "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                "tag": "v0.6.0"
+              }
+            ],
+            "cleanup": ["*"]
+        },  
+        {
+            "name" : "sudoku-solver",
+            "builddir" : true,
```

Comment:
Why is this enabled? Generally projects omit `builddir`.

Reply:
I used https://github.com/bottlesdevs/Bottles/blob/main/com.usebottles.bottles.yml#L396 and https://gitlab.com/TheEvilSkeleton/Upscaler/-/blob/main/io.gitlab.theevilskeleton.Upscaler.json#L57 as reference when making my manifest and both of them had `builddir`. 

Gonna remove it right away

Reply:
Done https://github.com/cyberphantom52/flathub/commit/c2dbed29b9bd4e55a7fe0f9f55214b6a1b9f3690

Reply:
it's always `true` for meson.

Reply:
> it's always `true` for meson.

Oh got it, i've removed it now.